src/StackExchange.Metrics/Handlers/StatsdMetricHandler.cs

```diff
@@ -13,10 +13,104 @@
 
 namespace StackExchange.Metrics.Handlers
 {
+    /// <summary>
+    /// Implements <see cref="IMetricHandler" /> by sending data to a statsd endpoint.
+    /// </summary>
+    public class StatsdMetricHandler : IMetricHandler
+    {
+        string _host;
+        ushort _port;
+        IMetricHandler _activeHandler;
+
+        /// <summary>
+        /// Constructs a new <see cref="StatsdMetricHandler" /> pointing at the specified host and port.
+        /// </summary>
+        /// <param name="host">
+        /// Host of a statsd endpoint.
+        /// </param>
+        /// <param name="port">
+        /// Port of a stats endpoint.
+        /// </param>
+        /// <remarks>
+        /// If the host or port are set to <c>null</c>/<c>0</c> this handler will be deactivated (every operation is a no-op).
+        /// Otherwise it sends data to a StatsD endpoint via UDP.
+        /// </remarks>
+        public StatsdMetricHandler(string host, ushort port)
+        {
+            Host = host;
```

Comment:
Probably wanna assign directly to the fields here so you don't double init the underlying handler when either host or port is not empty. `SignalFxMetricHandler` does the same:

https://github.com/StackExchange/StackExchange.Metrics/blob/main/src/StackExchange.Metrics/Handlers/SignalFxMetricHandler.cs#L33-L37